metadata_tools/common.py

```
@@ -0,0 +1,224 @@
+task_list = []
+
+#==========================================================================
+def task_source():
+    """Task source generator for cloud_tasks.
+    Args:
+        None.
+    Returns:
+        None
+    """
+    for task in task_list:
+        yield task
```

Ruff (0.14.10)

Replace `yield` over `for` loop with `yield from`

Replace with `yield from`

(UP028)